app-backup/snapper/snapper-0.5.4.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils
```

Comment:
only `prune_libtool_files` is used so you can inherit `ltprune` instead. But if you do `find "${D}" -name '*.la' -delete || die` instead you can drop inherit completely.